pkg/KubeArmorOperator/deployments/helm/templates/clusterrole-rbac.yaml

```diff
@@ -114,22 +114,15 @@ rules:
   - apps
   resources:
```

Comment:
@rksharma95 Please tell if my understanding is correct. The operator needs these perms so that it can pass them on to KubeArmor, right?

Reply:
yes you're right